src/Compilers/CSharp/Test/CommandLine/CSharpCommandLineTest.csproj

```diff
@@ -79,7 +79,7 @@
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.CodeAnalysis.Test.Resources.Proprietary, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+    <Reference Include="Microsoft.CodeAnalysis.Test.Resources.Proprietary, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
```

Comment:
It might be doing just that, but it is still breaking in future because the locally built version is v2.0.0.0 and the expected version (from the nuget) is v1.1.0.0.

I figure it was safer to remove the versioning specifics anyways since none of our other references have them.